app/tests/conftest.py

```diff
@@ -0,0 +1,4 @@
+from tests.fixtures.events import generic_event  # noqa: F401; pylint: disable=unused-variable
```

Comment:
We can investigate this, but generally I prefer disabling warning through config files (.flake8 or .mypy) or through the command line in the makefile rather than through comments in the code

eg. `pipenv run flake8 --ignore=E501,F401 app/;` but it will ignore unused import everywhere (not only in conftest file)

I think it is possible to ignore rules per file in the .flake8 file though

Reply:
I guess that because we are not using generic_exchange (and others) in this same conftest.py file, they are considered unused by pycharm, which makes sense.

I see two solutions:
* Make the imports in the test files that use them, directly (I prefer this one)
* See if pycharm can ignore this warning for conftest files (https://stackoverflow.com/questions/22951489/pycharm-noinspection-for-whole-file)